vars/provisionNodes.groovy

```diff
@@ -110,8 +110,8 @@ def call(Map config = [:]) {
     }
     // Prepare the node for daos/cart testing
     def provision_script = '''set -ex
-                              groupadd -g 1101 jenkins
-                              useradd -b /localhome -g 1101 -u 1101 jenkins
+                              groupadd -g 1101 jenkins || true
+                              useradd -b /localhome -g 1101 -u 1101 jenkins || true
```

Comment:
These will plow through an actual error to add the group or user.

Can we apply a more positive check for skip, like grepping the group and password files for the entry and not adding them if they exist?

Reply:
I can do that.